kolibri/utils/cli.py

```diff
@@ -333,11 +333,18 @@ def create_startup_lock(port):
         logger.warn(u"Impossible to create file lock to communicate starting process")
 
 
-@main.command(
-    cls=KolibriDjangoCommand,
-    help=u"Start the Kolibri process",
-    context_settings=dict(ignore_unknown_options=True, allow_extra_args=True),
-)
+def get_deprecated_daemon_arg(background):
```

Comment:
looks like you forgot to remove this function

Reply:
D'oh!